utilities/DateUtilities.ts

```diff
@@ -133,6 +133,21 @@ export class DateUtilities {
   ) {
     return arr.sort((a, b) => DateUtilities.gSorter(a, b, prop, desc));
   }
+
+  public static isWithin13Weeks(dateToCheck: Date, extra?: string): boolean | string {
+    //gets todays date
+    const now = new Date();
+    //gets the value of the startDate minus 13 weeks
+    const thirteenWeeksEarlier = new Date(dateToCheck.getTime() - (13 * 7 * 24 * 60 * 60 * 1000));
+
+    if (extra == "string")
```

Comment:
The `extra` variable is poorly named and the usage is a bit awkward, you *could* use a boolean instead but I think a separate function would be preferred in this case. Returning a string from a function named `isWithin13Weeks` is confusing.

I feel the `isWithin13Weeks` method itself is badly named too, the name is too tightly coupled to the implementation. If we wanted to change it to 90 days in the future we'd have to refactor the method name.

I think we should move this logic in to `CoJUtilities.ts` with something like the below

```typescript
private static SIGNABLE_OFFSET = 13 * 7 * 24 * 60 * 60 * 1000; // 13 weeks.

public static canBeSigned(startDate: Date): boolean {
  const now = new Date();
  const signableFrom = DateUtilities.getSignableFromDate(startDate);
  return now >= signableFrom;
}

public static getSignableFromDate(startDate: Date): Date {
  return new Date(startDate.getTime() - DateUtilities.SIGNABLE_OFFSET);
}
```